src/Stratis.Bitcoin.Features.PoA/PoABlockHeader.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NBitcoin;
+using NBitcoin.Crypto;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockHeader : BlockHeader
+    {
+        private BlockSignature federationSignature;
```

Comment:
hmm I don't like the name why not just call it `blockSignature`

Reply:
why not just have `public BlockSignature BlockSignature { get; set; }`?

Reply:
fixed